### ISSUE TITLE
Meetup Page improvement

### ISSUE THREAD

**Issue by nhessler** [ruby-conferences/ruby-conferences.github.io#632]
I'd like to propose a limit on how far in the future the website shows meetups.

Meetups are a bit different than Conferences in that there are so many and they happen so often. to that end, I think the meetup page is a bit over indexed in that if I'm looking for a meetup in the next 30-90 days it would be hard to find them as they are somewhere in the middle of the page as there are events currently posted through the end of the year.

Or maybe there is an option to add filtering or sections based on location and/or month?

to be clear, I think what's been done so far, is awesome! and I'm open to conversation, but just wanted to bring up something I noticed with the meetup page as a meetup organizer.

**Comment by marcoroth** [comment]
Yeah, you bring up a good point! I'm not opposed to limit it in a way. I think either a certain number of days, by month, or by the meetup itself could be good ways to reduce the amount of meetups shown 👍🏼

**Comment by marcoroth** [comment]
I'll re-open this issue, since I think we can still improve the page

**Comment by marcoroth** [comment]
Just merged https://github.com/ruby-conferences/ruby-conferences.github.io/pull/748 which should help with this as well.
